.github/workflows/edge.yml

```diff
@@ -437,20 +434,47 @@ jobs:
           git -c user.name='${{ env.GIT_NAME }}' -c user.email='${{ env.GIT_MAIL }}' commit -m "NGINX Ingress Controller - Edge Release (latest)"
           git push -u origin master
 
-  job-failure:
-    name: Report Failures
+  notify:
+    name: Notify
     runs-on: ubuntu-18.04
-    if: 
-      failure() && github.ref == 'refs/heads/master'
+    needs: release-helm
+    if: always() && github.ref == 'refs/heads/master'
     steps:
-      - name: Slack Notification
-        uses: rtCamp/action-slack-notify@master
+      - name: Workflow Status
+        id: check
+        uses: martialonline/workflow-status@v1
+      - name: Calculate SHA
+        id: commit
+        run: echo "::set-output name=sha::$(echo ${GITHUB_SHA} | cut -c1-7)"
+      - name: Send Notification
+        uses: 8398a7/action-slack@v3
```

Comment:
Just curious, what does this mean?

Reply:
It's a third party Action / plugin to trigger Slack notifications for failed builds.